### PR TITLE
[WIP] add log output to debug build fail on jenkins

### PR DESCRIPTION
**Pull request by evidolob**
This pull request to debug build failing on jectkins, do not accept it!!!

### CODE REVIEW

**Comment by skabashnyuk** [comment]
ok

**Comment by evidolob** [comment]
ci-build

File: plugins/plugin-maven/che-plugin-maven-server/src/main/java/org/eclipse/che/plugin/maven/server/MavenServerManager.java [hunk 1]
```diff
@@ -86,6 +86,7 @@ private ProcessExecutor createExecutor() {
         return () -> {
             JavaParameters parameters = buildMavenServerParameters();
             CommandLine command = parameters.createCommand();
+            LOG.debug("Creating new Executor for: " + parameters.getJavaExecutable());
```
Comment (benoitf) [inline plugins/plugin-maven/che-plugin-maven-server/src/main/java/org/eclipse/che/plugin/maven/server/MavenServerManager.java:89]: for log debug, we should not use string concatenation but object parameters

File: plugins/plugin-maven/che-plugin-maven-server/src/main/java/org/eclipse/che/plugin/maven/server/execution/ProcessHandler.java [hunk 2]
```diff
@@ -160,6 +161,7 @@ private void notifyOnTerminating() {
     }
 
     private void notifyOnText(String text, ProcessOutputType type) {
+        LOG.debug(type.name() + " " + text);
```
Comment (benoitf) [inline plugins/plugin-maven/che-plugin-maven-server/src/main/java/org/eclipse/che/plugin/maven/server/execution/ProcessHandler.java:164]: string concatenation shouldn't be used in DEBUG traces (give object as parameters and use kind of MessageFormat

File: plugins/plugin-maven/che-plugin-maven-server/src/main/java/org/eclipse/che/plugin/maven/server/rmi/RmiClient.java [hunk 3]
```diff
@@ -97,6 +97,7 @@ private void startProcess(Pair<Object, Object> key) {
         try {
             ProcessExecutor executor = getExecutor();
             handler = executor.execute();
+            LOG.debug("Maven Server JVM process executed: " + handler.toString());
```
Comment (benoitf) [inline plugins/plugin-maven/che-plugin-maven-server/src/main/java/org/eclipse/che/plugin/maven/server/rmi/RmiClient.java:100]: concatenation shouldn't be used for debug trace

File: plugins/plugin-maven/che-plugin-maven-server/src/test/java/org/eclipse/che/plugin/maven/server/rmi/MavenServerManagerTest.java [hunk 4]
```diff
@@ -53,7 +54,8 @@
 
 
     @BeforeMethod
-    public void setUp() throws Exception {
+    public void setUp(Method method) throws Exception {
+        System.out.println("TEST NAME:  "+method.getName() + "!!!!!!!!!!!!!!!!!");
```
Comment (benoitf) [inline plugins/plugin-maven/che-plugin-maven-server/src/test/java/org/eclipse/che/plugin/maven/server/rmi/MavenServerManagerTest.java:58]: I would say that even in a test println shouldn't be used

File: plugins/plugin-maven/maven-server/maven-server-api/src/main/java/org/eclipse/che/rmi/RmiServer.java [hunk 5]
```diff
@@ -33,9 +33,9 @@ protected static void start(Remote remote) throws RemoteException {
         if (RmiServer.remote != null) {
             throw new AssertionError("This server is already started!");
         }
-
         RmiServer.remote = remote;
 
+        System.err.println("Maven Server try to select port");
```
Comment (benoitf) [inline plugins/plugin-maven/maven-server/maven-server-api/src/main/java/org/eclipse/che/rmi/RmiServer.java:38]: is it an error message ?
we can't use Logger to print ?

File: plugins/plugin-maven/maven-server/maven-server-api/src/main/java/org/eclipse/che/rmi/RmiServer.java [hunk 6]
```diff
@@ -51,14 +51,16 @@ protected static void start(Remote remote) throws RemoteException {
             }
 
         }
-
+        System.err.println("Maven Server port selected: " + port);
```
Comment (benoitf) [inline plugins/plugin-maven/maven-server/maven-server-api/src/main/java/org/eclipse/che/rmi/RmiServer.java:54]: is it an error message ?
we can't use Logger to print ?

File: plugins/plugin-maven/maven-server/maven-server-api/src/main/java/org/eclipse/che/rmi/RmiServer.java [hunk 7]
```diff
@@ -51,14 +51,16 @@ protected static void start(Remote remote) throws RemoteException {
...
         Remote exportObject = UnicastRemoteObject.exportObject(remote, port);
         String exportName = remote.getClass().getSimpleName() + Integer.toHexString(exportObject.hashCode());
+        System.err.println("Maven Server remote object exported.");
```
Comment (benoitf) [inline plugins/plugin-maven/maven-server/maven-server-api/src/main/java/org/eclipse/che/rmi/RmiServer.java:57]: it should use DEBUG logger

File: plugins/plugin-maven/maven-server/maven-server-api/src/main/java/org/eclipse/che/rmi/RmiServer.java [hunk 8]
```diff
@@ -51,14 +51,16 @@ protected static void start(Remote remote) throws RemoteException {
...
         try {
             registry.bind(exportName, exportObject);
             String portName = port + "/" + exportName;
             System.out.println("Port/Name:" + portName);
             int twoMinutes = 2 * 60 * 1000;
+            System.err.println("Maven Server port and name printed");
```
Comment (benoitf) [inline plugins/plugin-maven/maven-server/maven-server-api/src/main/java/org/eclipse/che/rmi/RmiServer.java:63]: it should use DEBUG logger

**Comment by benoitf** [comment]
I've added WIP as it is only for jenkins :-)

**Pull request closed without merging**